tools/toascii/main.cpp

```diff
@@ -58,53 +49,19 @@ static void PrintUsage(const char* exe_name)
     GFXRECON_WRITE_CONSOLE("\nOptional arguments:");
     GFXRECON_WRITE_CONSOLE("  -h\t\t\tPrint usage information and exit (same as --help).");
     GFXRECON_WRITE_CONSOLE("  --version\t\tPrint version information and exit.");
+    GFXRECON_WRITE_CONSOLE("  --output <'stdout' or filepath>\t'stdout' or a working directory relative filepath to "
```

Comment:
The filepath doesn't have to be relative, the way you've coded it, it can be absolute as well.  Also this doesn't line up with the other options in the usage output and the output line is very long.  I suggest, like in `replay_options.h` to make this line up with the other options to do something like:
```
    GFXRECON_WRITE_CONSOLE("  --output file\t\t'stdout' or a path to a file to write JSON output");
    GFXRECON_WRITE_CONSOLE("               \t\tto. Default is the input filepath with \"gfxr\" replaced by \"txt\".");
```